### PR TITLE
Update test inspec profile URLs

### PR DESCRIPTION
**Pull request by alexpop**
### :nut_and_bolt: Description: What code changed, and why?

Remove dependency on Victoria's test inspec profiles.

### CODE REVIEW

**Review by srenatus — APPROVED** [review]
Thanks! :rocket:

File: components/automate-gateway/integration/license_usage_nodes_test.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func (suite *GatewayTestSuite) TestLicenseUsageNodes() {
 		Name:          "my job for aws-api node manager",
 		Tags:          []*common.Kv{},
 		Type:          "exec",
-		Profiles:      []string{"https://github.com/vjeffrey/try-inspec-aws-profile/archive/master.tar.gz"},
+		Profiles:      []string{"https://github.com/chef/automate/raw/master/components/compliance-service/test_data/inspec_profiles/test-aws-profile-2.0.0.tar.gz"},
```
Comment (srenatus) [inline components/automate-gateway/integration/license_usage_nodes_test.go:113]: I guess we'd be a little safer if this wasn't master, but a commit ref 🤔

Comment (alexpop) [reply components/automate-gateway/integration/license_usage_nodes_test.go:113]: I agree, will make the change tomorrow, thank you!

**Review by rickmarry — COMMENTED** [review]
thanks @alexpop

**Pull request merged**